kuma/users/tests/test_views.py

```diff
@@ -127,12 +127,13 @@ def tearDown(self):
     def _get_current_form_field_values(self, doc):
         # Scrape out the existing significant form field values.
         form = dict()
+        prefix = 'user-'
         for fn in ('username', 'email', 'fullname', 'title', 'organization',
                    'location', 'irc_nickname', 'bio', 'interests'):
-            form[fn] = doc.find('#user-edit *[name="user-%s"]' %
-                                fn).val()
-        form['country'] = 'us'
-        form['format'] = 'html'
+            form[prefix + fn] = doc.find('#user-edit *[name="user-%s"]' %
```

Comment:
nit: while we're in here, can we change `fn` to `fieldname` or `field_name` or something? 2-character variable names are only 50% less offensive than 1-character variable names. :smile: